main/typings.d.ts

```diff
@@ -1,4 +1,21 @@
+export type ActivitySource = 'Station' | 'Saturn';
+export type ActivityEventType = 'info' | 'error';
+
+export interface ActivityEvent {
+  type: ActivityEventType;
+  source: ActivitySource;
+  message: string;
+}
+
+export interface ActivityEntry extends ActivityEvent {
+  id: string;
+  timestamp: number;
+}
```

Comment:
I am not entirely happy with these two names. They are very similar, I think it's not immediately clear what the difference is between `ActivityEvent` and `ActivityEntry`.

Can we find better names?

One option that comes to my mind is to rename `ActivityEntry` to something like `PersistedActivityEvent` or `StoredActivityEvent`.

WDYT, @juliangruber ?